contributors.yml

```
@@ -1,6 +1,7 @@
 - 0xEddie
 - aarbi
 - abdallah-nour
+- Abdullah-Yirik
```

```suggestion
- xdaxer
```
You need to use your GitHub username